src/commands/releases.rs

```diff
@@ -515,7 +514,14 @@ fn execute_set_commits<'a>(
         }
         ctx.api.set_release_refs(&org, version, heads)?;
     } else {
-        println!("Could not determine any commits to be associated with a repo-based integration. Proceeding to find commits from local git tree.");
+        let default_count = matches
+            .value_of("initial-depth")
+            .unwrap_or("20")
+            .parse::<usize>()?;
+
+        if matches.is_present("local") {
```

Comment:
You log this only when `local` is present, yet the code runs every time

Reply:
I thought about it and realized that this message should only run during `--auto` bc it's saying that we couldn't find any **repo-based integrations**, which we don't check for during `--local`. 

Also, I ran it 'as is' with `--auto` and `--local` and it only showed up for `--local`.